servers/quarkus-common/src/main/java/org/projectnessie/quarkus/config/QuarkusBigTableConfig.java

```diff
@@ -21,6 +21,11 @@
 import java.util.Optional;
 import org.projectnessie.versioned.storage.bigtable.BigTableClientsConfig;
 
+/**
+ * When setting {@code nessie.version.store.type=BIGTABLE} which enables Google BigTable as the
+ * version store used by the Nessie server, the following configurations are applicable in
```

Comment:
Nit: I'd remove the `in combination with {@code nessie.version.store.type}` everywhere as it is a bit redundant.

Reply:
Yea - removed